src/server/async_disconnect_handler.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2016 CUBRID Corporation
```

Comment:
```suggestion
  *
  * Copyright 2016 CUBRID Corporation
```

Sorry for wrong suggestion...
I followed the license in schema_class_truncator.hpp

Reply:
I didn't check it too. Thank you. :)